### PR TITLE
Dao summary style

### PR DESCRIPTION
**Pull request by christopherdancy**
image

image

image

### CODE REVIEW

**Review by tbwebb22 — CHANGES_REQUESTED** [review]
Looks great. I like that you reversed the order of the proposals so the most recent is at the top. 

The only issues I see are:

1) The Open / Closed box at the top left of the proposal card styling should change depending on whether the proposal is open or closed.

2) The DAO details link, proposal count, and Add Proposal button are missing

File: src/components/DAOs/DAO/Proposals/ProposalCard.tsx [hunk 1]
```diff
@@ -11,17 +13,20 @@ function ProposalCard({
 
   return (
     <Link to={`proposals/${proposal.number}`}>
-      <div className="flex flex-col bg-gray-300 m-2 max-w-lg py-2 rounded-md">
-        <div className="flex flex-row mx-1">
-          <div className="mx-1">{proposal.stateString}</div>
-          <div className="mx-1">#{proposal.number}</div>
-          <div className="mx-1">
+      <ContentBox onHover="hover:border-2 hover:border-gold-300 hover:shadow-sm hover:shadow-gold-300">
```
Comment (adamgall) [inline src/components/DAOs/DAO/Proposals/ProposalCard.tsx:16]: This hover border makes all the cards jump around

File: src/components/ui/svg/Clock.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const Clock = () => {
   return (
     <svg
-      width="24"
-      height="24"
-      viewBox="0 0 24 24"
+      width="17"
+      height="17"
+      viewBox="0 0 24 20"
```
Comment (adamgall) [inline src/components/ui/svg/Clock.tsx:6]: This clock on Zeplin looks to be 24x24 px. What does it look like if you take the SVG as downloaded from Zeplin and replace all of this `svg` with it?

File: src/components/ui/ContentBox.tsx [hunk 3]
```diff
@@ -2,13 +2,15 @@ import ContentBoxTitle from "./ContentBoxTitle";
 
 const ContentBox = ({
   title,
+  onHover,
```
Comment (adamgall) [inline src/components/ui/ContentBox.tsx:5]: `onHover = ""`

like this

File: src/components/ui/ContentBox.tsx [hunk 4]
```diff
@@ -2,13 +2,15 @@ import ContentBoxTitle from "./ContentBoxTitle";
...
   children,
 }: {
   title?: string,
+  onHover?: string,
   children: React.ReactNode,
 }) => {
   return (
-    <div className="rounded-lg bg-gray-600 px-4 py-4 my-4">
+    <div className={`rounded-lg bg-gray-600 px-4 py-4 my-2 ${onHover}`}>
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/ContentBox.tsx:13]: However, I would rather see this hover functionality moved out of this ContentBox, and into another component (like right on the `ProposalCard` directly, probably).

File: src/components/DAOs/DAO/Proposals/ProposalCard.tsx (hunk 1 above)
Comment (christopherdancy) [reply src/components/DAOs/DAO/Proposals/ProposalCard.tsx]: fixed. added border and changes border color

File: src/components/DAOs/DAO/Proposals/ProposalsList.tsx [hunk 5]
```diff
@@ -5,8 +5,8 @@ function ProposalsList() {
   const [{ proposals }] = useDAOData();
 
   return (
-    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
-      {proposals?.map((proposal) => (
+    <div className="grid grid-cols-1 lg:grid-cols-2 2xl:grid-cols-3 lg:gap-4">
+      {proposals?.slice(0).reverse().map((proposal) => (
```
[the remark this answers is not in the document]
Comment (christopherdancy) [reply src/components/DAOs/DAO/Proposals/ProposalsList.tsx:9]: where to flip/reverse the proposal array

[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/DAOs/DAO/Proposals/ProposalsList.tsx:9]: but is `slice(0)` needed?

doesn't `proposals.reverse()` work on its own?

**Review by adamgall — APPROVED** [review]

**Pull request merged**